setup.py

```diff
@@ -53,6 +53,7 @@
         'prometheus': ['prometheus_client'],
         'dev': [
             'gunicorn',
+            'six',
```

Comment:
Should remove six as we aren't using it, as like Diego mentioned, this app is py3 only.